docs/pages/faq.mdx

```diff
@@ -63,6 +63,31 @@ are not logged in to Teleport or may not have access to a browser.
 Make sure that all Teleport components are at least at version 10.3.0. Older versions
 don't support the SFTP protocol, and it's enabled by default in `tsh` v11.0.0 and OpenSSH v9.0.
 
+## How do I extract certificates from a container running a distroless image?
```

Comment:
Would it make more sense to put this information in the Installation page where we introduce distroless images? Otherwise, it strikes me as a little unexpected to have a specific question like this right above something general like "How is Open Source different from Enterprise".

Otherwise, I think the wording is okay.

Reply:
Moved, and updated example to use `docker exec` rather than `kubectl exec`